### ISSUE TITLE
Time zone handling in queries causes confusion

### ISSUE THREAD

**Issue by daniel-thom** [dsgrid/dsgrid#192]
Consider this example: Query asks for comstock data to be transformed to StandardScenarios.

- StandardScenarios uses EST (no DST) and the time dimension query name is `time_est`.
- dsgrid produces timestamps that correspond to the correct absolute points in time.
- Spark records the timestamps in UTC in the Parquet file.
- When the user calls `spark.sql.DataFrame.show()` in the REPL, Spark converts the timestamps to the time zone of the JVM (which can be modified dynamically in the SparkSession).
- When the user calls `spark.sql.DataFrame.collect()` or `spark.sql.DataFrame.toPandas()`, the timestamps get converted to the time zone of the current computer.

1. Do we need to change something? Perhaps just document it?
2. Should we provide a procedure to convert the dataframe to Pandas, encode a time zone, and rewrite the Parquet file? (Pandas will encode time zone information in the Parquet metadata.)
3. Should we add the Pandas metadata into the Parquet metadata with the expectation that users will want it? This could be very confusing if person 1 uses Spark and person 2 uses Pandas.

**Comment by daniel-thom** [comment]
Assigning to @elainethale for clarification on the requirements. It's unlikely that we can implement any code for this in the current milestone.

**Comment by elainethale** [comment]
I agree that fixing this is out of scope for the current milestone. But I appreciate flagging this in some way as a key issue that will need fixing sooner rather than later.

**Comment by elainethale** [comment]
Dan's new opinion:
- DuckDB does not recognize time zones written in Parquet files by default Spark, because by default Spark uses int96 format, which is not standard.
- You can set a Spark flag to use something more standard. **So we should switch to that.**
